requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+pip==8.1.2
+bumpversion==0.5.3
+wheel==0.29.0
+watchdog==0.8.3
```

Comment:
The first 4 deps here are probably implicit, and don't need to be hardcoded in here?